docs/asciidoc/faq.adoc

```diff
@@ -206,4 +206,102 @@ processor.publishOn(scheduler1)
          .subscribe();
 ----
 
+== What is a good pattern for contextual logging? (MDC)
```

Comment:
Headings should be in title case, so this should be "What Is a Good Pattern for Contextual Logging? (MDC)".